projects/core/src/model/product-search.model.ts

```diff
@@ -31,9 +31,22 @@ export interface Facet {
   multiSelect?: boolean;
   name?: string;
   priority?: number;
-  topValues?: FacetValue[];
   values?: FacetValue[];
   visible?: boolean;
+
+  /**
+   * Indicates the top values that will be shown instantly. The top values can be
+   * controlled by business users per facet.
+   */
+  topValueCount: number;
```

Comment:
Should this be an optional value?

Reply:
I thought no, as I didn't implement conditional checks in the logic, and since the value is provided in the normalizer it's always there. That being said, changing my mind now, since the model is used for both source and target (backend and UI model), so it's going to fail in this case since the backend doen's provide this property.